package.json

```diff
@@ -86,8 +90,7 @@
     "varint-decoder": "~0.1.1"
   },
   "pre-push": [
-    "lint",
-    "test"
+    "lint"
```

Comment:
Can we please keep the pre-push testing. I think they run fast enough so that it is OK.

Reply:
Added that back.